api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSNAP.java

```diff
@@ -105,8 +109,8 @@ private void runPipelineCaffe(boolean useGPU) {
                 "other/tensor,dimension=(string)3:224:224:1,type=(string)float32,framerate=(fraction)0/1 ! " +
                 "tensor_filter framework=snap " +
                     "model=" + models[0].getAbsolutePath() + "," + models[1].getAbsolutePath() + " " +
-                    "input=3:224:224:1 inputtype=float32 inputname=data " +
-                    "output=1:1:1000:1 outputtype=float32 outputname=prob " +
+                    "input=3:224:224:1 inputtype=float32 inputlayout=NHWC inputname=data " +
```

Comment:
Q: if `inputlayout` is `NHWC`, the layout of input seems like `1:224:224:3`, not `3:224:224:1`.  Or, is there any underlying assumption on this?

Reply:
in snap subplugin, the dimension is reversed when invoking the model. (3:224:224:1 -> input shape in snap 1:224:224:3)

Reply:
Oh, got it.